core/drivers/crypto/stm32/stm32_cryp.c

```diff
@@ -1231,6 +1233,51 @@ TEE_Result stm32_cryp_update(struct stm32_cryp_context *ctx, bool last_block,
 	return res;
 }
 
+static TEE_Result stm32_cryp_reset(void)
+{
+	TEE_Result ret = TEE_SUCCESS;
```

Comment:
Other functions uses variable label `res` for `TEE_Result` values, prefer use `res` here also rathter than `ret`.
Ditto in `stm32_cryp_pm()` implementation.